<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>planningView</class>
 <widget class="QWidget" name="planningView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>688</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3">
   <item>
    <layout class="QVBoxLayout" name="layout" stretch="0,0,0">
     <item>
      <widget class="QLineEdit" name="searchLine">
       <property name="placeholderText">
        <string>Search ...</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <item>
        <widget class="QComboBox" name="monthBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="yearBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>37</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>37</height>
          </size>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="planningButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>37</height>
          </size>
         </property>
         <property name="text">
          <string>Plan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="activateButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>37</height>
          </size>
         </property>
         <property name="text">
          <string>Activate</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>37</height>
          </size>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exportButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>37</height>
          </size>
         </property>
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="table"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
